server/containers/images/gen-tags-from-rpm

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
OK, I'm seeing what you meant about duplication between the agent and server builds. Ugh.

Reply:
Yeah, once this is working, I'll refactor them.